hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.sample.yml

```diff
@@ -27,6 +27,19 @@ services:
       - 8082:8082
 
   Carlos:
+<<<<<<< HEAD
```

Comment:
Merge gone wrong ?

Reply:
I had disabled the examples. merging bits only until things work.